Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Adapted from: https://github.com/elasticdog/tiddlywiki-docker/blob/master/5/Dockerfile
+
+ARG NODE_LTS_VERSION=18
+
+FROM node:${NODE_LTS_VERSION}-alpine
+
+ARG TIDDLYWIKI_VERSION=5.2.7
+ARG TIDDLYWIKI_DATA_DIR=/tiddlywiki
+ARG TIDDLYWIKI_PORT=8080
+
+# https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#handling-kernel-signals
+RUN apk add --no-cache tini
+RUN npm install -g tiddlywiki@${TIDDLYWIKI_VERSION}
```

Comment:
I wonder if it would be better to not have a dependency on npm here. Since this Dockerfile is being included with the git repository all the source code is already available, I would expect the docker image to be built using this local source (i.e. using the docker COPY command) rather than from whatever happens to be in npm.

Especially if some github actions CI job were created to build the docker image. It doesn't seem as useful to checkout the entire TW git repository, but then use none of the source code except for the Dockerfile?

But the result from using `COPY` would require invoking it as `nodejs tiddlywiki.js` rather than just `tiddlywiki`. Maybe that is ok or maybe there is some simple way around this.

I'm just thinking out loud here. Any other thoughts?